signature-specification.md

```diff
@@ -0,0 +1,228 @@
+# Signature Specification
+
+This document describes how Notary v2 signatures are created and stored.
+The document consists of following sections:
+* **[Storage](#storage)**: Describes how signatures are stored in OCI registry
+* **[Signature Envelope](#signature-envelope)**: Describes how signatures are created.
+
+
+## Storage
+This section describes how Notary v2 signatures are stored in the OCI Distribution conformant registry.
+Notary v2 uses [ORAS artifact manifest](https://github.com/oras-project/artifacts-spec/blob/main/artifact-manifest.md) to store the signature in the repository. The signature artifact manifest consists of a signature type, a reference to the manifest of the artifact being signed, a blob referencing to the signature, and a collection of annotations.
+<img src="media/signature-specification.jpg" width="550" height="450">
```

Comment:
A picture helps with clarity. 
A few clarifications:
- The manifest that contains the signature is of type `application/vnd.cncf.oras.artifact.manifest.v1+json`
- The top box suggests it can only sign an OCI image manifest. I'd suggest changing this to simply be "Manifest". The `subject`, by which the signature is associated with can be any manifest type ([oci image manifest](https://github.com/opencontainers/image-spec/blob/main/manifest.md), [oci image index](https://github.com/opencontainers/image-spec/blob/main/image-index.md), [oras-artifact manifest](https://github.com/oras-project/artifacts-spec/blob/main/artifact-manifest.md)) which means Notary v2 can sign any artifact from container images to anything else that follows the conventions in [OCI Artifacts authors guidance](https://github.com/opencontainers/artifacts/blob/master/artifact-authors.md), a multi-arch manifest of images, or other artifacts that use the ORAS Artifact manifest for things like SBoMs, scan results and other reference types
- Suggest taking the top box within the two lower boxes and make them a header on the box. I was reading it as the content, not the description of the content.

Reply:
incorporated feedback